pallets/stream-payment/src/lib.rs

```diff
@@ -145,6 +155,13 @@ pub mod pallet {
         /// Provide interaction with assets.
         type Assets: Assets<Self::AccountId, Self::AssetId, Self::Balance>;
 
+        type Currency: Inspect<Self::AccountId, Balance = Self::Balance>
```

Comment:
Add comment that explain this is only for the deposit and should not be confused with Assets, which are assets the streams can pay in.